src/components/Login/Login.scss

```
@@ -0,0 +1,117 @@
+@import '../../styles/variables';
+.loginScreen {
+  display: flex;
+  flex-direction: column;
+  width: 45%;
+  height: 100vh;
+
+  background-color: $mainBackground;
+
+  .loginWrapper {
+    width: 80%;
+    margin-top: 38px;
+    margin-left: 30px;
+    .login {
+      margin-top: 10px;
+      font-size: 30px;
+    }
+
+    .loginEmail {
+      width: 100%;
+      height: 36px;
+      margin-top: 67px;
+
+      border-top-width: 0;
+      border-right-width: 0;
+      border-bottom-width: 1;
+      border-left-width: 0;
+
+      background: transparent;
+      &::placeholder {
+        font-size: 25px;
+      }
+    }
+    .nonfalse {
+      display: none;
+    }
+    .false {
+      display: block;
+      color: red;
+    }
+    .pwWrapper {
```

```suggestion
    .nonfalse {
      display: none;
    }
    
    .false {
      display: block;
      color: red;
    }
    
    .pwWrapper {
```

가독성을 위해 스타일 끼리 한칸씩 띄워주세요!